index.js

```diff
@@ -13,6 +13,8 @@ const QUIET_FLAG = 'quiet';
 const TARGET_BUILD_FLAG = 'target_build';
 const VERIFY_FLAG = 'verify';
 
+// If flags are added, deleted, or modified, UPDATE README.
```

Comment:
Aha I now see the comment here, so maybe including the help message in README is fine.